src/tests/unit/test_remove.py

```diff
@@ -84,9 +103,10 @@ def test_remove_child_dict(self):
             "2": {"name": "Nested", "description": "", "type": "basic_blueprint"},
         }
 
-        self.document_service.remove(Address("$1.nested", "testing"))
-        assert self.storage["1"].get("nested") is None
-        assert self.storage.get("2") is None
+        self.assertRaises(ValidationException, self.document_service.remove, Address("$1.nested", "testing", "dmss"))
+        # TODO Test fails due to modelling error in blueprint. Cannot delete required attribute
```

Comment:
would be nice to also add a new issue to the "incoming" tab in the board to fix modeling errors in the blueprints used in this test_remove.py also 👍

Reply:
Ok, I will add it to the board!